ansible/roles/baselayout/tasks/partials/repo/centos7.yml

```diff
@@ -6,11 +6,18 @@
     state: present
 
 - name: centos7 | install ius
-  when: "arch != 'arm64'"
+  when: "arch != 'arm64' and arch != 'ppc64'"
   yum:
     name: "https://centos{{ ansible_distribution_major_version }}.iuscommunity.org/ius-release.rpm"
     state: present
 
+- name: centos7 | install sclo7 devtoolset-6
```

Comment:
I have #1765 which is for x64 centos7 and it's going to conflict with this, one of us probably needs to change, or we need to add arch exclusions. Mine is just:

```
 - name: centos7 | install scl for devtoolset-6
  when: "arch != 'arm64'"
  yum:
    name: centos-release-scl
    state: present
```

and I'm not sure about the arm64 exclusion there, we might be able to get rid of the sclo aarch64 bits further down this file but it'll need some testing (which I don't have time for right now!)

@sam-github might a simple `yum install centos-release-scl` work for ppc64 too?

Reply:
I'm not so sure what these incantations do, but I replaced mine with your example above, and am trying it on a fresh host. I'll see if it works.

Reply:
Unfortunately, no, I don't think that repo works for ppc64le:
```
TASK [baselayout : install packages] **************************************************
failed: [release-osuosl-centos7-ppc64_le-1] (item=ccache,gcc-c++,devtoolset-6,sudo,git,devtoolset-6-libatomic-devel,python3) => {"changed": false, "msg": "Failure talking to yum: Cannot find a valid baseurl for repo: centos-sclo-sclo/ppc64le", "package": "ccache,gcc-c++,devtoolset-6,sudo,git,devtoolset-6-libatomic-devel,python3"}                
failed: [release-osuosl-centos7-ppc64_le-1] (item=automake,bash,libtool) => {"changed": false, "msg": "Failure talking to yum: Cannot find a valid baseurl for repo: centos-sclo-sclo/ppc64le", "package": "automake,bash,libtool"}
```

Reply:
I'm reverting back to what I did, but with `when: "arch == 'ppc64'"`.